server/command.go

```diff
@@ -600,6 +640,11 @@ func getAutocompleteData() *model.AutocompleteData {
 	webhookAdd.AddTextArgument("[Optional] token: Secret for webhook. Defaults to token used in plugin's settings.", "[token]", "")
 	webhook.AddCommand(webhookAdd)
 
+	webhookBuild := model.NewAutocompleteData(commandBuild, "owner/[repo] reference", "trigger the pipeline")
+	webhookAdd.AddTextArgument("Group or Project path: includes user or group name with optional slash project name", "owner[/repo]", "")
+	webhookAdd.AddTextArgument("reference: reference/branch name for which the pipeline needs to be triggered", "reference", "")
```

Comment:
Should this be
```suggestion
	webhookBuild.AddTextArgument("Group or Project path: includes user or group name with optional slash project name", "owner[/repo]", "")
	webhookBuild.AddTextArgument("reference: reference/branch name for which the pipeline needs to be triggered", "reference", "")
```
?

Reply:
seems to be fixed here: https://github.com/lufty-demansol/mattermost-plugin-gitlab/blob/master/server/command.go#L563